roslibrust/Cargo.toml

```diff
@@ -31,6 +31,7 @@ thiserror = "1.0.31"
 [dev-dependencies]
```

Comment:
Can we change edition to 2021 above here, so we have consistency across workspace